src/internal/components/chart-legend/index.tsx

```diff
@@ -41,33 +42,27 @@ function ChartLegend<T>({
 
   const highlightedSeriesIndex = findSeriesIndex(series, highlightedSeries);
 
-  const highlightLeft = () => {
+  const highlightInlineStart = () => {
     const currentIndex = highlightedSeriesIndex ?? 0;
     const nextIndex = currentIndex - 1 >= 0 ? currentIndex - 1 : series.length - 1;
     segmentsRef.current[nextIndex]?.focus();
   };
 
-  const highlightRight = () => {
+  const highlightInlineEnd = () => {
     const currentIndex = highlightedSeriesIndex ?? 0;
     const nextIndex = currentIndex + 1 < series.length ? currentIndex + 1 : 0;
     segmentsRef.current[nextIndex]?.focus();
   };
 
-  const handleKeyPress = (event: React.KeyboardEvent) => {
+  const handleKeyPress = (event: React.KeyboardEvent<HTMLElement>) => {
     if (event.keyCode === KeyCode.right || event.keyCode === KeyCode.left) {
       // Preventing default fixes an issue in Safari+VO when VO additionally interprets arrow keys as its commands.
       event.preventDefault();
 
-      switch (event.keyCode) {
```

Comment:
@pan-kot I removed the return syntax here. Everything works fine, I just want you to double check it.